sdk/servicebus/azure-servicebus/samples/async_samples/test_examples_async.py

```diff
@@ -402,7 +402,7 @@
 @pytest.mark.asyncio
 async def test_async_sample_queue_send_receive_batch(live_servicebus_config, standard_queue):
```

Comment:
Better change test_examples_async.py and test_examples.py to have no pytest stuffs

Reply:
Per our OOB discussion:  Given that we're going to be changing the structure of resource creation significantly for T2, and are already deprecating most of the fixtures in preference for preparers, I'm leaning towards putting this as a TODO and a line item in our release epic to ensure we've cleaned up these samples with respect to the new code and don't have to do two passes over it.